function_execute.go

```diff
@@ -0,0 +1,83 @@
+package slack
```

Comment:
Please ensure that the WebAPI functions accept a context as the first function parameter and rename this `FunctionCompleteErrorContext`. You can create a second function called `FunctionCompleteError` that explicitly sets `contact.Background()` if you'd like. 

You can look at [ListEventAuthorizationsContext](https://github.com/slack-go/slack/blob/master/apps.go#L31) and [ListEventAuthorizations](https://github.com/slack-go/slack/blob/master/apps.go#L25) for an example of how to implement it.